core/App/components/listItems/NotificationListItem.tsx

```diff
@@ -219,13 +227,89 @@ const NotificationListItem: React.FC<NotificationListItemProps> = ({ notificatio
     })
   }, [notificationType])
 
+  const getContainerStyle = (type: InfoBoxType) => {
+    switch (type) {
```

Comment:
Can we combine these into a single object?

Reply:
> Can we combine these into a single object?

I agree, we could use a dict or something if that works. Something like:
```javascript
const getContainerStyle = {
    InfoBoxType.Info: {
        backgroundColor: ColorPallet.notification.info,
        borderColor: ColorPallet.notification.infoBorder,
    },
    ...
}
```